cli/src/alluxio.org/cli/env/env.go

```diff
@@ -224,7 +224,7 @@ func checkAndSetJava(envVar *viper.Viper) error {
 	// check if java is available via `PATH` using `which`
 	whichJavaPath, err := exec.Command("which", "java").Output()
 	if err == nil {
-		envVar.Set(ConfJava.EnvVar, string(whichJavaPath))
+		envVar.Set(ConfJava.EnvVar, strings.Trim(string(whichJavaPath), "\n"))
```

Comment:
```suggestion
		envVar.Set(ConfJava.EnvVar, strings.TrimSpace(string(whichJavaPath)))
```
trims all whitespace characters including newline